databasehandler.py

```diff
@@ -19,6 +19,19 @@ class Card(Base):
     def __repr__(self) -> str:
         return self.__dict__.__str__()
 
+    def to_json(self):
+        return {
+            'id': self.id,
+            'original': self.word_original,
```

Comment:
original and meaning are a bit vague in general. What language is original? What language is meaning?

Reply:
This also depends on if you want to extend it to many languages or just swedish->english or viceversa